src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java

```diff
@@ -103,6 +103,24 @@
  * &lt;module name="EmptyBlock"/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public class Test {
+ *   private void emptyLoop() {
+ *     for (int i = 0; i &lt; 10; i++) {
+ *       // violation
```

Comment:
This comment should be on the line where violation occurred. Please fix all below cases as well.

Reply:
fixed